umm-spec-lib/src/cmr/umm_spec/migration/characteristics_data_type_migration.clj

```diff
@@ -0,0 +1,73 @@
+(ns cmr.umm-spec.migration.characteristics-data-type-migration
```

Comment:
General comment: I feel like `migration` is a wrong word to use here. It is more of a `normalization` than `migration`.

Reply:
It's written for the migration purpose, used by the version_migration.clj. and it just happened that it could be used by other part of the code as well. It is consistent with the collection-progress-migration.

Reply:
I would like to think it as migration code happens to use the normalization code rather than the other way around.

Reply:
okay. Done.